### PR TITLE
Remove InteractiveWindow

### PR DESCRIPTION
**Pull request by tmat**
Moved to https://github.com/dotnet/interactive-window

### CODE REVIEW

**Comment by tmat** [comment]
ETA test failures expected.

**Comment by tmat** [comment]
@dotnet/roslyn-infrastructure @jasonmalinowski @shyamnamboodiripad @brettfo @jmarolf Please review, especially VSIX related changes.

**Comment by tmat** [comment]
@KevinH-MS

**Comment by tmat** [comment]
@jasonmalinowski @dotnet/roslyn-infrastructure

**Comment by tmat** [comment]
test windows_release_unit32_prtest  please

**Comment by tmat** [comment]
@KevinH-MS

File: src/EditorFeatures/CSharpTest/project.json [hunk 1]
```diff
@@ -4,6 +4,10 @@
             "version": "14.3.25407",
             "suppressParent": "all"
         },
+        "RoslynDependencies.Microsoft.VisualStudio.Text.Internal": {
+           "version": "14.3.25407",
+           "suppressParent": "all"
+        },
```
Comment (KevinH-MS) [inline src/EditorFeatures/CSharpTest/project.json]: Is this necessary?

Comment (tmat) [inline src/EditorFeatures/CSharpTest/project.json]: Seems to be. Build fails otherwise

Comment (KevinH-MS) [inline src/EditorFeatures/CSharpTest/project.json]: Surprising, but I accept that there are mysteries in the universe... :smile:

Perhaps we were embedding the interop types in the InteractiveWindow.

**Comment by KevinH-MS** [comment]
:shipit:

File: src/Deployment/source.extension.vsixmanifest [hunk 2]
```diff
@@ -29,15 +29,6 @@
                 Location="|VisualStudioSetup;VSIXContainerProjectOutputGroup|"
                 Id="|VisualStudioSetup;VSIXIdentifierProjectOutputGroup|" />
     
-    <Dependency d:ProjectName="VisualStudioInteractiveWindow"
```
Comment (jasonmalinowski) [inline src/Deployment/source.extension.vsixmanifest]: Should this dependency (in some modified form, obviously) have been left?

File: src/EditorFeatures/CSharpTest/project.json (hunk 1 above)
Comment (jasonmalinowski) [inline src/EditorFeatures/CSharpTest/project.json]: IIRC the InteractiveWindow uses some types from this, so it doesn't surprise me.

**Pull request merged**